cmd/regex_format.go

```diff
@@ -291,3 +283,12 @@ func formatEndOfFile(lines []string) []string {
 	// to the others by newline
 	return append(lines[:eof+1], "")
 }
+
+func checkStandardHeader(lines []string) bool {
+	if len(lines) > 2 &&
```

Comment:
Why not use the constant here?

Reply:
The constant is a `string`, and here we have `[]string`

Reply:
Yes, but I would convert the array to a string here and check against the constant (or split the constant into lines) instead of duplicating the constant content.

Reply:
Done.